WalletWasabi.Fluent.Desktop/Program.cs

```diff
@@ -26,17 +27,20 @@ public class Program
 		// This is only needed to pass CrashReporter to AppMainAsync otherwise it could be a local variable in Main().
 		private static CrashReporter CrashReporter;
 
+		private static TerminateService TerminateService;
```

Comment:
This line adds: `Warning	CS8618	Non-nullable field 'TerminateService' must contain a non-null value when exiting constructor. Consider declaring the field as nullable.`

Reply:
ACK.